packages/suite/src/views/settings/SettingsDevice/DisplayRotation.tsx

```diff
@@ -21,12 +22,20 @@ interface DisplayRotationProps {
 export const DisplayRotation = ({ isDeviceLocked }: DisplayRotationProps) => {
     const dispatch = useDispatch();
     const { device } = useDevice();
-    const currentRotation = device?.features?.display_rotation;
 
-    if (typeof currentRotation !== 'number') {
+    // features.display_rotation cannot be used to determine support because can be defined for devices not supporting rotation (e.g. T2B1).
+    const isSupported =
+        device?.features != undefined &&
+        [DeviceModelInternal.T2T1, DeviceModelInternal.T3T1].includes(
```

Comment:
nit: `const DEVICES_SUPPORTING_DISPLAY_ROTATION = [DeviceModelInternal.T2T1, DeviceModelInternal.T3T1]` at the beginning of the file

Reply:
5d50504df3cd18ff649a169961a0c0190a6821dd